bin/rails

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
+
```

Comment:
@sinpai what do you think about skipping this `frozen_string_literal: true`? I mean you can switch this cop off and remove all of `frozen_string_literal: true` 😉

Reply:
ok 👍  It came from suggested .rubocop.yml from chat. A lot of files was changed by `rubocop -a`. I'll fix it.

Reply:
@sinpai `frozen_string_literal: true` makes rails faster, and probably should be used where possible. So, it's your decision here - use or not to use `frozen_string_literal`